client/client.go

```diff
@@ -1752,9 +1755,14 @@ OUTER:
 	}
 }
 
-// updateNode triggers a client to update its node copy if it isn't doing
+// updateNode updates the Node copy and triggers the client to send the updated
+// Node to the server. This should be done while holding the configLock lock.
```

Comment:
Maybe specify that the caller needs to hold the configLock

Reply:
Already states that.